contributors/guide/first-contribution.md

```diff
@@ -19,57 +19,96 @@ description: |
+Check out the [list of SIGs] for contact information.
 
-#### SIG structure
+Understanding how to interact with SIGs is an important part of contributing 
+to Kubernetes. 
+
+### SIG structure
 
 A SIG is an open, community effort.
-Anybody is welcome to jump into a SIG and begin fixing issues, critiquing design proposals and reviewing code.
-SIGs have regular [video meetings](https://kubernetes.io/community/) which everyone is welcome to.
-Each SIG has a slack channel, meeting notes, and their own documentation that is useful to read and understand.
 
-There is an entire SIG ([sig-contributor-experience](/sig-contributor-experience/README.md)) devoted to improving your experience as a contributor.
-Contributing to Kubernetes should be easy.
-If you find a rough edge, let us know! Better yet, help us fix it by joining the SIG; just
-show up to one of the [bi-weekly meetings](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit).
+Anybody is welcome to jump into a SIG and begin fixing issues, critiqe design 
+proposals, and review code.
+
+SIGs have regular [video meetings] which everyone is welcome to attend.
 
-#### Find a SIG that is related to your contribution
+Each SIG has a Slack channel, meeting notes, and their own documentation that 
+is useful to read and understand.
 
-Finding the appropriate SIG for your contribution and adding a SIG label will help you ask questions in the correct place and give your contribution higher visibility and a faster community response.
+There is an entire SIG ([sig-contributor-experience] devoted to 
+improving your experience as a contributor.
 
-For Pull Requests, the automatically assigned reviewer will add a SIG label if you haven't done so. See [Open A Pull Request](#open-a-pull-request) below.
+If you have an idea for improving the contributor experience, please consider
+attending one of the Contributor Experience SIG's [bi-weekly meetings].
 
-For Issues, we are still working on a more automated workflow.
-Since SIGs do not directly map onto Kubernetes subrepositories, it may be difficult to find which SIG your contribution belongs in.
-Here is the [list of SIGs](/sig-list.md) so that you can determine which is most likely related to your contribution.
+### Find a SIG that is related to your contribution
+
+Finding the appropriate SIG for your contribution and adding a SIG label will 
+help you ask questions in the correct place and give your contribution higher 
+visibility and a faster community response.
+
+For Pull Requests, the automatically assigned reviewer will add a SIG label 
+if you haven't already done so. 
+
+For Issues, please note that the community is working on a more automated workflow.
+
+Since SIGs do not directly map onto Kubernetes subrepositories, it may be 
+difficult to find which SIG your contribution belongs in.
+
+Review [list of SIGs](/sig-list.md) to determine which SIG is most likely related 
+to your contribution.
```

Comment:
```suggestion
Review [list of SIGs][sl] to determine which SIG is most likely related to your
contribution.
```
This could be updated to use the reference link shared with the others 👍